core/config/entity.go

```diff
@@ -105,3 +105,40 @@ func checkValid(conf *SentinelConfig) error {
 	}
 	return nil
 }
+
+func (entity Entity) AppName() string {
```

Comment:
Would using a pointer receiver make more sense?

Reply:
If the function logic does not modify the Entity,  it does not seem necessary to use pointers.

Reply:
In future versions, scenarios might exist that read configurations in the slot chain.
This copy operation is not acceptable.